src/org/dokuwiki/translatorBundle/Tests/Services/Language/LocalTextTest.php

```diff
@@ -32,7 +32,7 @@ public function testLicenceSurroundedByEmptyCommentlines() {
 <?php
 
 /**
- * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
+ * @license    GPL 3 (http://www.gnu.org/licenses/gpl.html)
```

Comment:
No, we need to stay at 2! We might need to change the URL to https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html

Reply:
Sure, this is a unit test. It tests if licenses as defined by language file author are maintained. Maybe this not most handy example..

Reply:
Ahh, I missed that this is a test... looked like a template in the diff.